test/integration/util_remediation_test.go

```diff
@@ -28,7 +28,7 @@ const (
 	kicsEngine                  = "engine"
 	similarityIDValue           = "9574288c118e8c87eea31b6f0b011295a39ec5e70d83fb70e839b8db4a99eba8"
 	resultFileInvalidValue      = "./"
-	invalidEngineMessage        = "Please verify if engine is installed and running"
+	invalidEngineMessage        = "Please verify if engine is installed"
```

Comment:
Same as above, is it possible to access a public constant in the regular code from the tests?